lerobot/scripts/visualize_image_transforms.py

```diff
@@ -0,0 +1,46 @@
+from pathlib import Path
+
+import hydra
+from torchvision.transforms import ToPILImage
+
+from lerobot.common.datasets.lerobot_dataset import LeRobotDataset
+from lerobot.common.datasets.transforms import make_image_transforms
+
+to_pil = ToPILImage()
+
+
+def main(cfg, output_dir=Path("outputs/image_transforms")):
+    dataset = LeRobotDataset(cfg.dataset_repo_id, image_transforms=None)
+
+    output_dir = Path(output_dir) / Path(cfg.dataset_repo_id.split("/")[-1])
+    output_dir.mkdir(parents=True, exist_ok=True)
+
+    # Get first frame of 1st episode
+    first_idx = dataset.episode_data_index["from"][0].item()
+    frame = dataset[first_idx][dataset.camera_keys[0]]
+    to_pil(frame).save(output_dir / "original_frame.png", quality=100)
+
+    transforms = ["brightness", "contrast", "saturation", "hue", "sharpness"]
+
+    # Apply each single transformation
+    for transform_name in transforms:
+        for t in transforms:
+            if t == transform_name:
+                cfg.image_transforms[t].weight = 1
+            else:
+                cfg.image_transforms[t].weight = 0
+
+        transform = make_image_transforms(cfg.image_transforms)
+        img = transform(frame)
+        to_pil(img).save(output_dir / f"{transform_name}.png", quality=100)
+
```

Comment:
Could you print the output directory?

Reply:
Fixed https://github.com/huggingface/lerobot/pull/234/commits/540fb9c55061d9c84a40298f25c83e78932826e3